docs/videos/healthcare-and-life-sciences/README.md

```diff
@@ -1,6 +1,10 @@
 # Healthcare and Life Science Blueprint
 
-The Healthcare and Life Science (HCLS) [blueprint](./hcls-blueprint.yaml) in
+> [!NOTE]
+> This document uses SlurmGCP v6 version of hcls blueprint. If you want to
+> use SlurmGCP v5 version, please refer [blueprint](./hcls-blueprint.yaml).
```

Comment:
s/please refer [blueprint]/please refer to this [blueprint]/

Reply:
Done.